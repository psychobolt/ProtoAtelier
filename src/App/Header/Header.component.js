// @flow
import React from 'react';

import Link from './Link';

type Props = {
  centered: boolean,
  style: {},
};

const Header = ({ centered, style = {} }: Props) => (
  <x-tabs style={style} centered={centered || null}>
    <Link to={[{ to: '/' }, { to: '/box', default: true }]}>
      Box
    </Link>
    <Link to="/sphere">
      Sphere
    </Link>
  </x-tabs>
);

export default Header;
