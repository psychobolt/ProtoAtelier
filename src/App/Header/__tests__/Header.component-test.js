import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header.component';

test('component <Header /> should render correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
