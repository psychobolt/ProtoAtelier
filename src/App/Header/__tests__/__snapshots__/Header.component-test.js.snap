// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component <Header /> should render correctly 1`] = `
<x-tabs
  centered={null}
  style={Object {}}
>
  <Link
    to={
      Array [
        Object {
          "to": "/",
        },
        Object {
          "default": true,
          "to": "/box",
        },
      ]
    }
  >
    Box
  </Link>
  <Link
    to="/sphere"
  >
    Sphere
  </Link>
</x-tabs>
`;
